docs/src/main/sphinx/security/opa-access-control.md

```diff
@@ -212,6 +218,74 @@ The `targetResource` is used in cases where a new resource, distinct from the on
   }
 }
 ```
+## RowFilters 
+
+Raw filters allow you to denied a group of users to see specifics rows in a table, based on an OPA policy.
+
+for these kind of policies to work, make sure to set `opa.policy.row-filters-uri` in the opa-plugin configuration
+
+here is an example bellow
+```
+            package trino
+            import future.keywords.in
+            import future.keywords.if
+            import future.keywords.contains
+
+            default allow := true
+
+            table_resource := input.action.resource.table
+            is_admin {
+              input.context.identity.user == "admin"
+            }
+
+            rowFilters contains {"expression": "user_type <> 'customer'"} if {
+                not is_admin
+                table_resource.catalogName == "sample_catalog"
+                table_resource.schemaName == "sample_schema"
+                table_resource.tableName == "restricted_table"
+            }
+```
+the format expected by the plugin is a result (here "rowFilters") that contains an array of "{"expressions":"clause"} where each expression is basicaly a where clause.
+ 
+
+## Column masking
+
+Column masking allow you to denied access to a particular column instead of the entiere tables from your OPA policy.
+
+for these kind of policies to work, make sure to set `opa.policy.column-masking-uri` in the opa-plugin configuration
+
+here is an example bellow
+
+```
+       package trino
+            import future.keywords.in
+            import future.keywords.if
+            import future.keywords.contains
+
+            default allow := true
+
+            column_resource := input.action.resource.column
+            is_admin {
+              input.context.identity.user == "admin"
+            }
+
+            columnMask := {"expression": "NULL"} if {
+                not is_admin
+                column_resource.catalogName == "sample_catalog"
+                column_resource.schemaName == "sample_schema"
+                column_resource.tableName == "restricted_table"
+                column_resource.columnName == "user_phone"
+            }
+
+            columnMask := {"expression": "'****' || substring(user_name, -3)"} if {
+                not is_admin
+                column_resource.catalogName == "sample_catalog"
+                column_resource.schemaName == "sample_schema"
+                column_resource.tableName == "restricted_table"
+                column_resource.columnName == "user_name"
+            }
+```
+
```

Comment:
Similarly to row filtering, I would mention:

- Unlike row filtering, only _a single_ column mask may be returned for a given column
- The same "identity" field may be returned to evaluate column masks under a different identity

Reply:
+1, proposal added